bundles/CoreBundle/Resources/config/pimcore/default.yaml

```diff
@@ -57,6 +57,7 @@ twig:
         # this is only here for compatibility/dev reasons and may be removed later
         container: '@service_container'
         pimcore_csrf: '@Pimcore\Bundle\AdminBundle\Security\CsrfProtectionHandler'
+        pimcore_nonce: '@Pimcore\Bundle\AdminBundle\Security\ContentSecurityPolicyHandler'
```

Comment:
Should we maybe call the variable `pimcore_csp`, as it is more than just the nonce?

Reply:
done